### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by sidvishnoi**
Installed dependabot, it started sending a lot of PRs (noise!). Doing this run manually, then will enable dependabot again.

Snyk didn't send any PRs :disappointed: 

Closes #2883
Closes #2884
Closes #2885 
Closes #2886

### CODE REVIEW

File: src/core/markdown.js [hunk 1]
```diff
@@ -53,6 +53,10 @@ const gtEntity = /&gt;/gm;
 const ampEntity = /&amp;/gm;
 
 class Renderer extends marked.Renderer {
+  constructor() {
+    super();
```
Comment (saschanaz) [inline src/core/markdown.js:57]: Why do we need this? Looks like a complete redundancy.

Comment (sidvishnoi) [reply src/core/markdown.js:57]: Not sure either, but lint started shouting (see previous commit)

Comment (sidvishnoi) [reply src/core/markdown.js:57]: Wait. We do need to extend this using `super()`, right?

Comment (sidvishnoi) [reply src/core/markdown.js:57]: That's weird. Calling `renderer: new Renderer(undefined)` works :thinking:

Comment (saschanaz) [reply src/core/markdown.js]:
```
> npm run lint

> respec@25.8.0 lint C:\Users\Kagami\Documents\GitHub\respec
> tsc -p src/jsconfig.json && eslint .

src/core/markdown.js:90:15 - error TS2554: Expected 1 arguments, but got 0.

90     renderer: new Renderer(),
                 ~~~~~~~~~~~~~~

  node_modules/marked/src/Renderer.js:11:15
    11   constructor(options) {
                     ~~~~~~~
    An argument for 'options' was not provided.


Found 1 error.
```

TypeScript is getting an incorrect inferred type. Could you try installing `@types/marked`?

Comment (sidvishnoi) [reply src/core/markdown.js]: Samesies. Just installed :smile:

File: package.json [hunk 2]
```diff
@@ -21,41 +21,42 @@
...
     "karma-firefox-launcher": "^1.3.0",
     "karma-jasmine": "^3.1.1",
-    "karma-jasmine-html-reporter": "^1.5.3",
-    "karma-mocha": "^1.3.0",
+    "karma-jasmine-html-reporter": "^1.5.4",
+    "karma-mocha": "^2.0.1",
     "karma-mocha-reporter": "^2.2.5",
     "karma-safari-launcher": "^1.0.0",
-    "marked": "^0.8.2",
+    "marked": "^1.1.0",
```
Comment (marcoscaceres) [inline package.json:49]: this is exciting...

File: package.json [hunk 3]
```diff
@@ -21,41 +21,42 @@
...
+    "rollup": "^2.10.4",
     "rollup-plugin-alias": "^2.2.0",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-terser": "^5.3.0",
     "serve": "^11.3.0",
-    "typescript": "^3.8.3",
-    "webidl2": "^23.11.0"
+    "typescript": "^3.9.2",
+    "webidl2": "^23.12.1"
```
Comment (marcoscaceres) [inline package.json:59]: this is also exciting! :)

**Comment by sidvishnoi** [comment]
Enabling dependabot now :robot:

**Comment by marcoscaceres** [comment]
With these deps, might be worth a point release?

**Pull request merged**